plugins/georeplication/transactions.go

```diff
@@ -324,6 +333,129 @@ func txnGeorepConfigFilegen(c transaction.TxnCtx) error {
 			return err
 		}
 	}
+	return nil
+}
+
+func txnSSHKeysGenerate(c transaction.TxnCtx) error {
+	var volname string
+	var err error
+	var args []string
+
+	if err = c.Get("volname", &volname); err != nil {
+		return err
+	}
+
+	secretPemFile := path.Join(
+		config.GetString("localstatedir"),
+		"geo-replication",
+		"secret.pem",
```

Comment:
It may not work since sshd picks first key from authorized_keys file. Tar command will fail if same key is used. I think we can change gsyncd C binary to accept tar similar to rsync command.

Reply:
We need to add tar support to gsyncd shell. It's good for now.